c2h/include/c2h/catch2_test_helper.h

```diff
@@ -27,29 +27,25 @@
 
 #pragma once
 
```

Comment:
This should at least include the cccl config so that we have access to `_CCCL_NV_DIAG_SUPPRESS`